peachjam/views/generic_views.py

```diff
@@ -56,6 +56,16 @@ def get_context_data(self, **kwargs):
         courts = []
         natures = []
         # Initialize facet data values
+        if self.model in [GenericDocument, LegalInstrument, Legislation]:
```

Comment:
Can we now simply do this in all cases, since all document models have natures?

Reply:
Even if they're potentially None?

Reply:
Yes we can. Changed.